server/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "start": "node server.js"
+    "start": "nodemon server.js"
```

Comment:
start should always be node. Nodemon is for development only and not suitable for production.
Create a new script called "start-dev" or "dev" and use nodemon there.